### DISCUSSION TITLE
supporting rust

### DISCUSSION THREAD

**Discussion by arashatt** [jdhao/nvim-config#273] [Q&A]
Do you have any idea what is the best way to add rust analyzer?
I have difficulty integrating rust support in this configuration. There is no good documentation about how to add [rust analyzer](https://rust-analyzer.github.io/manual.html#vimneovim) with lua.

**Comment by jdhao** [comment]
I think you should check the official doc [here](https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#rust_analyzer), then you can add its configuration to [here](https://github.com/jdhao/nvim-config/blob/main/lua/config/lsp.lua), just check how other lsp is configured.
